### ISSUE TITLE
Client (426) Scanning Malicious in Windows Defender

### ISSUE THREAD

**Issue by Chirishman** [hydrusnetwork/hydrus#791]
## Environment

Hydrus version: 426 - Executable
Platform: Windows
OS specifics: Server 2019

## What happens

Client.exe is popping in Windows Defender and SecureAge APEX and Windows Defender is reporting **Trojan:Win32/Wacatac.D!ml**

## Steps to reproduce

1. https://www.virustotal.com/gui/file/512b9607c86fff5481711cb94354c340e6ea5ba2deca4065f1c16ccb93d1bbab/detection

## Desired outcome

Probably this is a false positive and should be reported as such, looks like the dotnet crew are having a similar problem with some of their templates dotnet/runtime#35167

**Comment by Zweibach** [comment]
It's not a Hydrus issue, nothing dev can do about this to the best of my knowledge. Issue will be left open so people don't go creating dupes.

**Comment by bbappserver** [comment]
You can provide your own exclusion and tell microsoft they're wrong, but yes this is a windows defender issue.

Be careful though, if you did not download hydrus from an official source, it might genuinely just be malware.

Information on this detection
https://howtofix.guide/trojanwin32-wacatac-dml-2/

Where to obtain an official releases
https://github.com/hydrusnetwork/hydrus/releases

```
cat Hydrus.Network.426.-.Windows.-.Extract.only.zip | openssl dgst -sha256
(stdin)= 4ffef1ae7b8a8f0e6b6f56ac3cdd6ee882248c79a877c404714c3badf8d026e1
```

```
clamscan "Hydrus.Network.426.-.Windows.-.Extract.only.zip" "Hydrus Network"
Hydrus.Network.426.-.Windows.-.Extract.only.zip: OK
Hydrus Network/Qt5QmlModels.dll: OK
Hydrus Network/Qt5WebSockets.dll: OK
Hydrus Network/help my client will not boot.txt: OK
Hydrus Network/Qt5VirtualKeyboard.dll: OK
Hydrus Network/Qt5Charts.dll: OK
Hydrus Network/VCRUNTIME140.dll: OK
Hydrus Network/server.py: OK
Hydrus Network/Qt5Pdf.dll: OK
Hydrus Network/win32security.pyd: OK
Hydrus Network/python37.dll: OK
Hydrus Network/_elementtree.pyd: OK
Hydrus Network/Qt5Qml.dll: OK
Hydrus Network/_hashlib.pyd: OK
Hydrus Network/requirements.txt: OK
Hydrus Network/python3.dll: OK
Hydrus Network/Qt5Svg.dll: OK
Hydrus Network/win32api.pyd: OK
Hydrus Network/win32ui.pyd: OK
Hydrus Network/Qt5Network.dll: OK
Hydrus Network/win32wnet.pyd: OK
Hydrus Network/_ssl.pyd: OK
Hydrus Network/client.py: OK
Hydrus Network/win32file.pyd: OK
Hydrus Network/libssl-1_1.dll: OK
Hydrus Network/pythoncom37.dll: OK
Hydrus Network/d3dcompiler_47.dll: OK
Hydrus Network/client.exe: OK
Hydrus Network/sqlite3.dll: OK
Hydrus Network/_overlapped.pyd: OK
Hydrus Network/libcrypto-1_1.dll: OK
Hydrus Network/win32event.pyd: OK
Hydrus Network/MSVCP140_1.dll: OK
Hydrus Network/test.py: OK
Hydrus Network/mpv-1.dll: OK
Hydrus Network/_asyncio.pyd: OK
Hydrus Network/pylzma.cp37-win_amd64.pyd: OK
Hydrus Network/base_library.zip: OK
Hydrus Network/server.exe: OK
Hydrus Network/_lzma.pyd: OK
Hydrus Network/_multiprocessing.pyd: OK
Hydrus Network/Qt5Gui.dll: OK
Hydrus Network/pywintypes37.dll: OK
Hydrus Network/pyexpat.pyd: OK
Hydrus Network/pyside2.abi3.dll: OK
Hydrus Network/libEGL.dll: OK
Hydrus Network/Qt5Widgets.dll: OK
Hydrus Network/_cffi_backend.cp37-win_amd64.pyd: OK
Hydrus Network/VCRUNTIME140_1.dll: OK
Hydrus Network/_sqlite3.pyd: OK
Hydrus Network/shiboken2.abi3.dll: OK
Hydrus Network/Qt5DBus.dll: OK
Hydrus Network/_queue.pyd: OK
Hydrus Network/Qt5Core.dll: OK
Hydrus Network/Qt5Quick.dll: OK
Hydrus Network/opengl32sw.dll: OK
Hydrus Network/_socket.pyd: OK
Hydrus Network/win32gui.pyd: OK
Hydrus Network/Readme.txt: OK
Hydrus Network/opencv_videoio_ffmpeg440_64.dll: OK
Hydrus Network/_yaml.cp37-win_amd64.pyd: OK
Hydrus Network/win32process.pyd: OK
Hydrus Network/server.exe.manifest: OK
Hydrus Network/license.txt: OK
Hydrus Network/win32pipe.pyd: OK
Hydrus Network/_ctypes.pyd: OK
Hydrus Network/win32pdh.pyd: OK
Hydrus Network/client.pyw: OK
Hydrus Network/client.exe.manifest: OK
Hydrus Network/libopenblas.PYQHXLVVQ7VESDPUVUADXEVJOBGHJPAY.gfortran-win_amd64.dll: OK
Hydrus Network/libGLESv2.dll: OK
Hydrus Network/_bz2.pyd: OK
Hydrus Network/_win32sysloader.pyd: OK
Hydrus Network/win32evtlog.pyd: OK
Hydrus Network/select.pyd: OK
Hydrus Network/MSVCP140.dll: OK
Hydrus Network/mfc140u.dll: OK
Hydrus Network/_decimal.pyd: OK
Hydrus Network/unicodedata.pyd: OK
Hydrus Network/win32trace.pyd: OK

----------- SCAN SUMMARY -----------
Known viruses: 8716268
Engine version: 0.102.2
Scanned directories: 1
Scanned files: 80
Infected files: 0
Data scanned: 117.71 MB
Data read: 407.13 MB (ratio 0.29:1)
Time: 111.289 sec (1 m 51 s)
```

How to add an exclusion once you're sure this was erroneous. 
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-antivirus/antivirus-false-positives-negatives

**Comment by Chirishman** [comment]
@bbappserver Yeah, I installed it via the [chocolatey package repository](https://chocolatey.org/packages/hydrus-network) and I've doublechecked the installer checksum manually just to be sure. ([package source](https://github.com/AeliusSaionji/chocopkgs/tree/master/hydrus-network))

Actually if you look at the VirusTotal results link in my original post you'll see that Microsoft is no longer flagging the file though "SecureAge APEX" still is. This can probably be closed since M$ appears to have fixed the false positive.
